Packs/Genians/pack_metadata.json

```diff
@@ -2,9 +2,9 @@
     "name": "Genians",
     "description": "Use the Genians integration to block IP using assign tag and unassign tag.",
     "support": "partner",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
     "author": "Genians",
-    "url": "https://docs.genians.com/release/en/",
+    "url": "",
```

Comment:
@mchasepan What about this: https://www.genians.com/resources-overview/

Reply:
ok, approved.

Reply:
done